crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -145,19 +150,29 @@ impl ActionHandler for BridgeLockAction {
     }
 }
 
-/// returns the length of a serialized `Deposit` message.
-pub(crate) fn get_deposit_byte_len(deposit: &Deposit) -> u128 {
-    use prost::Message as _;
-    let raw = deposit.clone().into_raw();
-    raw.encoded_len() as u128
+/// Returns a modified byte length of the deposit event. Length is calculated with reasonable values
+/// for all fields except `asset` and `destination_chain_address`, ergo it may not be representative
+/// of on-wire length.
+pub(crate) fn calculate_base_deposit_fee(deposit: &Deposit) -> Option<u128> {
+    deposit
+        .asset
+        .display_len()
+        .checked_add(deposit.destination_chain_address.len())
+        .and_then(|var_len| DEPOSIT_BASE_FEE.checked_add(var_len as u128))
```

Comment:
It would be incredibly odd if we couldn't convert a `usize` to a `u128`. But for the sake of consistency, can you `u128::try_from(u128).expect("converting a usize to a u128 should work on any currently existing machine")`? This is for consistency.